src/components/dialog/dialog/readme.md

```diff
@@ -0,0 +1,131 @@
+The `sbb-dialog` component provides a way to present content on top of the app's content.
+It offers the following features:
+
+- creates a backdrop for disabling interaction below the modal;
+- disables scrolling of the page content while open;
+- manages focus properly by setting it on the first focusable element;
+- can host a [sbb-dialog-actions](/docs/components-sbb-dialog-actions--docs) component in the footer;
+- has a close button, which is always visible;
+- can display a back button next to the title;
+- adds the appropriate ARIA roles automatically.
+
+```html
+<sbb-dialog>
+  <sbb-dialog-title slot="title">Title</sbb-dialog-title>
```

Comment:
I'd never explicitely write the slot in the examples as it is assigned automatically